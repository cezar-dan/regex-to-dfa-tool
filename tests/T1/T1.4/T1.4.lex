 
SPACE
0
0,' ',1
1,' ',2
1

\n
NEWLINE
0
0,'\n',1
1,'\n',2
1

 01
PATTERN1
0
0,' ',1
0,'0',1
0,'1',2
2,' ',3
2,'0',1
2,'1',1
3,' ',1
3,'0',4
3,'1',1
4,' ',1
4,'0',1
4,'1',1
4

 01
PATTERN2
0
0,' ',1
0,'0',2
0,'1',3
1,' ',2
1,'0',2
1,'1',2
3,' ',2
3,'0',4
3,'1',2
4,' ',1
4,'0',2
4,'1',3
1

 01
PATTERN3
0
0,' ',1
0,'0',2
0,'1',2
1,' ',2
1,'0',3
1,'1',2
3,' ',2
3,'0',4
3,'1',2
4,' ',2
4,'0',2
4,'1',5
5,' ',6
5,'0',2
5,'1',2
6,' ',2
6,'0',2
6,'1',2
6

 01
PATTERN4
0
0,' ',1
0,'0',1
0,'1',2
2,' ',1
2,'0',4
2,'1',1
3,' ',1
3,'0',1
3,'1',2
4,' ',1
4,'0',1
4,'1',5
5,' ',3
5,'0',1
5,'1',1
3

01
PATTERN5
0
0,'0',1
0,'1',2
1,'0',3
1,'1',4
2,'0',1
2,'1',2
4,'0',3
4,'1',3
4