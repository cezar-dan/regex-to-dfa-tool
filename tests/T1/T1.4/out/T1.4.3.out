PATTERN2 101010 
PATTERN1 1 0
SPACE  
PATTERN1 1 0